net/unbound/Makefile

```diff
@@ -37,9 +37,9 @@ define Package/unbound
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=IP Addresses and Names
+  USERID:=unbound=553:unbound=553
```

Comment:
Hi, @EricLuehrsen , no need to rush when there are doubts ;)

By "fresh install" did you mean sysupgrade with old config preserved?  If that is the case, then the issue does exist in 15.05.1 but fixed in both OpenWrt/LEDE `master` branch with `package/base-files/files/etc/uci-defaults/13_fix_group_user`.  Archived discussion on this topic is at https://lists.openwrt.org/pipermail/openwrt-devel/2016-April/041160.html

In any case, I don't see the difference between using specific ids and the other way.  That dnsmasq using specific uid/gid is not a workaround of anything about uid/gid not being added correctly.